Services/Interfaces/web_interface/models/backtesting.py

```diff
@@ -354,20 +354,38 @@ def stop_data_collector():
     return success, message
 
 
-def create_snapshot_data_collector(exchange_id, start_timestamp, end_timestamp):
+def create_snapshot_data_collector(exchange_id, start_timestamp, end_timestamp, profile_id=None):
     exchange_manager = trading_api.get_exchange_manager_from_exchange_id(exchange_id)
+    symbols = trading_api.get_trading_symbols(exchange_manager)
+    time_frames = trading_api.get_relevant_time_frames(exchange_manager)
+    if profile_id is not None:
+        profile = profiles_model.get_profile(profile_id)
+        tentacles_setup_config = profiles_model.get_tentacles_setup_config_from_profile(profile)
+        strategies = configuration_model.get_config_activated_strategies(tentacles_setup_config)
+        time_frames = [
+            tf
+            for tf in configuration_model.get_traded_time_frames(
+                exchange_manager, strategies=strategies, tentacles_setup_config=tentacles_setup_config
+            ) or (commons_enums.TimeFrames.ONE_MINUTE,)
```

Comment:
Shouldn't we pass the default value in params instead of setting `commons_enums.TimeFrames.ONE_MINUTE`?